LoopKit/RemoteDataService.swift

```diff
@@ -118,5 +125,9 @@ public extension RemoteDataService {
     func uploadSettingsData(_ stored: [StoredSettings], completion: @escaping (_ result: Result<Bool, Error>) -> Void) {
         completion(.success(false))
     }
```

Comment:
No support for validation by default. The services can choose to implement this, like the NightscoutService will.